doc/oper.xml

```diff
@@ -962,6 +962,105 @@ gap> DigraphJoin(gr, gr2);
+    digraph with vertex set <C>Cartesian(DigraphVertices(G), DigraphVertices(H))</C> 
+    such that there is an edge from <C>[u, u']</C> to <C>[v, v']</C> iff <C> u = v </C> 
+    and there is an edge from <C>u'</C> to <C>v'</C> in <C>H</C> or 
+    <C> u' = v'</C> and there is an edge from <C>u</C> to <C>v</C> in <C>G</C>.
+    Due to technical reasons, the digraph returned by <C>DigraphCartesianProduct</C> has 
+    vertex set <C>[1 .. DigraphNrVertices(G)*DigraphNrVertices(H)]</C> instead. The vertex
+    pair can be somewhat regained by using <Ref Attr="DigraphCartesianProductProjections"/>.
+    <P/>
+
+    As the Cartesian product is associative, the Cartesian product of a collection 
+    of digraphs <A>gr1</A>, <A>gr2</A>, <C>...</C> is computed in the obvious 
+    fashion. 
+    <P/>
+
+    Note that previously set <Ref Oper="DigraphVertexLabels"/> will be lost.
+
+<Example><![CDATA[
+gap> gr := ChainDigraph(4);
+<immutable chain digraph with 4 vertices>
+gap> gr2 := CycleDigraph(3);
+<immutable cycle digraph with 3 vertices>
+gap> gr3 := DigraphCartesianProduct(gr, gr2);
+<immutable digraph with 12 vertices, 21 edges>
+gap> IsIsomorphicDigraph(gr3, 
+> Digraph([[2, 5], [3, 6], [4, 7], [8], 
+>          [6, 9], [7, 10], [8, 11], [12],
+>          [10, 1], [11, 2], [12, 3], [4]]));
+true
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="DigraphDirectProduct">
+<ManSection>
+  <Func Name="DigraphDirectProduct" Arg="gr1, gr2, ..." Label="for an arbitrary 
+   non-zero number of digraphs"/>
+  <Func Name="DigraphDirectProduct" Arg="list" Label="for a list of digraphs"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    In the first form, if <A>gr1</A>, <A>gr2</A>, etc. are digraphs, then
+    <C>DigraphDirectProduct</C> returns their direct product.
+    <P/>
+
+    In the second form, if <A>list</A> is a non-empty list of digraphs,
+    then <C>DigraphDirectProduct</C> returns the direct product of 
+    the digraphs contained in the list.
+    <P/>
+
+    Mathematically, the direct product of two digraphs <C>G</C>, <C>H</C> is a 
+    digraph with vertex set <C>Cartesian(DigraphVertices(G), DigraphVertices(H))</C> 
+    such that there is an edge from <C>[u, u']</C> to <C>[v, v']</C> iff there is an edge from
+    <C>u</C> to <C>v</C> in <C>G</C> and an edge from <C>u'</C> to <C>v'</C> in <C>H</C>. 
+    Due to technical reasons, the digraph returned by <C>DigraphDirectProduct</C> has 
+    vertex set <C>[1 .. DigraphNrVertices(G)*DigraphNrVertices(H)]</C> instead. The vertex
+    pair can be somewhat regained by using <Ref Attr="DigraphDirectProductProjections"/>.
+    <P/>
+ 
+    As the direct product is associative, the direct product of a collection 
+    of digraphs <A>gr1</A>, <A>gr2</A>, <C>...</C> is computed in the obvious 
+    fashion. 
+    <P/>
+
+    Note that previously set <Ref Oper="DigraphVertexLabels"/> will be lost.
```

Comment:
This sentence doesn't seem correct, having looked at the code. Please describe what the code does - you do actually set the vertex labels! And this can typically be used to work out the correspondence between vertices of the individual digraphs, and the vertices of the product - so perhaps this corrected sentence, and `"The vertex pair can be somewhat regained by using <Ref Attr="DigraphDirectProductProjections"/>."` can be combined somehow.

Analogous comment for the Cartesian product.